todo_app/templates/index.html

```diff
@@ -11,7 +11,23 @@ <h1 class="display-4">To-Do App</h1>
     <div class="col-md-12">
       <h2>Items</h2>
       <ul class="list-group mb-4">
-        <!-- List the todo items here -->
+        <body> 
+          <ul> 
+            {% for item in items %}
+            <li>{{ item.title }}</li>
+            {% endfor %}
+          </ul> 
+        </body> 
+        <form action="" method = "POST">
+          <div>
+            
+            <label for="title">Todo Title: </label>
+            <input type="text" id ="title" name="title" placeholder = "Enter Todo...">
```

Comment:
`required` is another useful attribute - it would prevent use submitting a blank todo.